packages/mimikatz.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Stop'
+$name = "Mimikatz"
+$path = Join-Path ${Env:TOOL_LIST_DIR} 'Password Attacks'
+
+$toolsDir = Join-Path "${Env:RAW_TOOLS_DIR}" $name
```

Comment:
Like above, you'll likely want `VM-Uninstall` (see: https://github.com/mandiant/VM-Packages/blob/main/packages/common.vm/tools/vm.common/vm.common.psm1#L473).

Here's an example of it being used: https://github.com/mandiant/VM-Packages/blob/main/packages/gobuster.vm/tools/chocolateyinstall.ps1